pages/common/hg-clone.md

```diff
@@ -0,0 +1,23 @@
+# hg clone
+
+> Create a copy of an existing repository in a new directory.
+
+- Clone a repository to a specified directory:
+
+`hg clone {{remote_repository_source}} {{destination_path}}`
+
+- Clone a repository on a specific branch:
+
+`hg clone --branch {{branch}} {{remote_repository_source}}`
+
+- Clone a repository with a clean working directory:
```

Comment:
What does "clean working directory" mean ? Does it mean you have to be inside a directory which already has the code ? Or it should be an empty directory ?

Reply:
Yeh, that makes sense it's not very clear.

It just means that it only creates the `.hg` directory and doesn't check out any files.